R/step-group.R

```diff
@@ -1,17 +1,30 @@
-step_group <- function(parent, groups = parent$groups) {
+step_group <- function(parent, groups = parent$groups, keyby = parent$keyby) {
```

Comment:
Would you mind also renaming this `keyby` argument to arrange? I think it will make the code a bit easier to read.

Reply:
Done!